opentelemetry-api/src/common.rs

```diff
@@ -498,7 +498,7 @@ impl InstrumentationLibrary {
             name: name.into(),
             version: version.map(Into::into),
             schema_url: schema_url.map(Into::into),
-            attributes,
+            attributes: attributes.unwrap_or_default(),
```

Comment:
I kept the `attributes` parameter as `Option<Vec<KeyValue>>` here because it's more idiotmatic to express `None` for "I have no attributes to provide" than to require the api user to actually give up ownership of an empty vector.  IOW, `Option` better expresses "optional" parameters... but we don't need to store it as such, hence the `unwrap_or_default()`